FordLabs/retroquest
# Combine active and completed column items

## Overview
Combine active and completed column items so that the columns

Connects N/A


## Testing Instructions
Nothing should have changed in the UI

1) Run the app and run cypress tests. Ensure they pass.
2) Run the app and manually go through the app. Ensure everything works as usual.
